CodeConverter/Shared/VisualBasicCompiler.cs

```diff
@@ -52,6 +52,8 @@ private static VisualBasicCompilation CreateVisualBasicCompilation()
 
         public static VisualBasicCompilationOptions CreateCompilationOptions(string rootNamespace = null)
         {
+            // Caution: The simplifier won't always remove imports unused by the code (e.g. System.Data)
+
```

Comment:
One case that causes them not to be removed is when they can't be resolved - could this be to do with removing the reference to SqlCommand?

Reply:
I will check again tomorrow, but I don't think so because on .net472 (framework version of tests assembly) sqlcommand and datarow are within the same assembly. I suspect it's a name/overload conflict being resolved by the compiler between system.data and another assembly (possibly system.linq?) since if I copy the output of a failing test which gets an extra system.data using, the compiler itself doesn't trigger the usual unused using warnings.

Reply:
It was indeed caused by overloading resolution. I've updated the comment for clarification.